lib/mpl_toolkits/mpl_parsave/__init__.py

```diff
@@ -0,0 +1,2 @@
+
+__all__ = ["mplparsave"]
```

Comment:
I would get rid of this package and just make a `parsave` module under `mpl_toolkits`